firmware/steering/kia_soul_ps/test_steering_control.h

```diff
@@ -0,0 +1,35 @@
+/************************************************************************/
+/* The MIT License (MIT)                                                */
+/* =====================                                                */
+/*                                                                      */
+/* Copyright (c) 2016 PolySync Technologies, Inc.  All Rights Reserved. */
+/*                                                                      */
+/* Permission is hereby granted, free of charge, to any person          */
+/* obtaining a copy of this software and associated documentation       */
+/* files (the “Software”), to deal in the Software without              */
+/* restriction, including without limitation the rights to use,         */
+/* copy, modify, merge, publish, distribute, sublicense, and/or sell    */
+/* copies of the Software, and to permit persons to whom the            */
+/* Software is furnished to do so, subject to the following             */
+/* conditions:                                                          */
+/*                                                                      */
+/* The above copyright notice and this permission notice shall be       */
+/* included in all copies or substantial portions of the Software.      */
+/*                                                                      */
+/* THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,      */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES      */
+/* OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND             */
+/* NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT          */
+/* HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,         */
+/* WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING         */
+/* FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR        */
+/* OTHER DEALINGS IN THE SOFTWARE.                                      */
+/************************************************************************/
+
+#ifndef TEST_STEERING_CONTROL_H
+#define TEST_STEERING_CONTROL_H
+
+bool test_pid_values( PID &pid_params,
```

Comment:
Formatting is odd here also.  You might want to have a description of the function in the include file.  Not sure this is relevant in this case as this is all open source.